### DISCUSSION TITLE
date_locale

### DISCUSSION THREAD

**Discussion by petergCA** [marcokreeft87/formulaone-card#78] [General]
Would you mind pointing me to the options for this customization?

**Comment by marcokreeft87** [comment]
In the end it uses the second example on the top of the page of https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat.
But if you don't pass anything it uses the locale from HA

**Comment by petergCA** [comment]
Thanks for this - I'll leave blank and see if it translates correctly!
